que_5.py

```diff
@@ -0,0 +1,20 @@
+Numbers = [9, 4, 8, 10, 2, 4, 8, 3, 14, 4, 8]
+n = 12
+first = 0
+
+list_1 = []
+for first in range(0,len(Numbers)-1):
+    for last in range(first +1 ,len(Numbers)-1):
+
+   
+        if (Numbers[first] + Numbers[last]) == n:
```

Comment:
Learn about the itertools, you can use the combinations from the itertools to complete this.